### DISCUSSION TITLE
Welcome to MobileAutomationFramework Discussions!

### DISCUSSION THREAD

**Discussion by shankybnl** [shankybnl/MobileAutomationFramework#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chetanJTH** [comment]
Hi. I am Chetan Thummar and I am an automation tester.
I mostly try to find out the framework-related solutions on this platform.
I report issues whatever I try and we learn together.
